doc/sphinx/package_service_mapping.json

```diff
@@ -993,7 +998,7 @@
   },
   "azure-mgmt-storagesync": {
     "category": "Management",
-    "service_name": "Storage",
+    "service_name": "Storage Sync",
```

Comment:
this + the namespaces in `_package_service_mapping` combine to make certain that `azure-mgmt-web` actually generates documentation.

Reply:
@zikalino can you take a look and see if this looks right? [`azure-mgmt-web`](https://azuresdkdocs.blob.core.windows.net/$web/python/azure-mgmt-web/0.43.1/azure.mgmt.web.html#submodules )